docs/scanners/gitleaks.md

```diff
@@ -0,0 +1,43 @@
+# [Gitleaks](https://github.com/zricethezav/gitleaks)
+
+The [Gitleaks Scanner](https://github.com/zricethezav/gitleaks) tool for finding secrets within a directory or Git repo.
+
+## Configuration
+
+Gitleaks currently always runs with the `--recursive` and `--format json` options turned on.
+In addition, you can configure the additional options below.
+For more information on Gitleaks configuration, see the docs [here](https://github.com/zricethezav/gitleaks#usage-and-options)
```

Comment:
Your scanner only supports the Gitleaks config options below, correct?
If so, can you clarify that and say not all config options in the link above are supported?

Reply:
Also, can you specify whether each option is required or optional?

Reply:
It doesn't look like gitleaks has an option to exclude dirs, correct?